### PR TITLE
Fixed visualization of extruded 1D meshes

### PR DESCRIPTION
**Pull request by najlkin**
This PR fixes visualization of 1D meshes, which are incorrectly recognized as coarse meshes of quadratures, since 1D meshes are backed up before the extrusion (to be used with different representations of quadratures).

Reported by: @vladotomov 

TODO:
- [x] Add 1D test

### CODE REVIEW

**Comment by najlkin** [comment]
@vladotomov , added a 1D test and waiting on approval of GLVis/data#6 .

**Review by tzanio — APPROVED** [review]

**Pull request merged**
